doc/sphinx/credits.rst

```diff
@@ -1199,25 +1199,25 @@ Credits: version 8.7
 and cleanups of the internals of the system along with a few new features. The
 main user visible changes are:
 
-- New tactics: variants of tactics supporting existential variables eassert,
-  eenough, etc... by Hugo Herbelin. Tactics extensionality in H and
-  inversion_sigma by Jason Gross, specialize with ... accepting partial bindings
+- New tactics: variants of tactics supporting existential variables :tacv:`eassert`,
+  :tacv:`eenough`, etc... by Hugo Herbelin. Tactics ``extensionality in H`` and
+  :tacv:`inversion_sigma` by Jason Gross, ``specialize with ...`` accepting partial bindings
```

Comment:
Same here.

Reply:
Replacing ``:tacv:`inversion_sigma` `` by ``:tacn:`inversion_sigma` `` did not work. Despite `proof-engine/tactics.rst` containing `.. tacv:: inversion_sigma`, the script complains: `tacn reference target not found: inversion_sigma`. What am I missing?

Reply:
The other issue is that, by default, tactic variants do not get an index, unless you give them a `:name:`. @cpitclaudel I think this was a request from @maximedenes, right?

Reply:
Right.